Fix bugs

vincetiu8 opened the pull request:
This commit removes the bug where enemies speed up after a tower has been placed. It also removes some bullet curve so towers can shoot enemies traveling up to 2000 speed.

Fixes issues #9 and #11.

vincetiu8 committed:
Fix bugs

This commit removes the bug where enemies speed up
after a tower has been placed. It also removes some
bullet curve so towers can shoot enemies travelling
up to 2000 speed.

Graphcalibur commented:
Looks good but I don't want to merge it yet since the starting tile seems to be yellow now instead of green like before

vincetiu8 committed:
Change starting tile back to green

One of the fixes for the pathing in this bugfix added the starting tile to
the list of nodes, meaning it appeared in yellow instead of green. This
commit changes it back to green for visibility purposes.
